lib/perl/Genome/Model/Command/Define/ImportedReferenceSequence.t

```diff
@@ -17,10 +17,11 @@ if (Genome::Sys->arch_os ne 'x86_64') {
     plan skip_all => 'requires 64-bit machine';
 }
 
+my $version = 1;
 my $cmd_class = 'Genome::Model::Command::Define::ImportedReferenceSequence';
 use_ok($cmd_class);
 
-my $data_dir = File::Temp::tempdir('ImportedAnnotationTest-XXXXX', CLEANUP => 1, TMPDIR => 1);
+my $data_dir = get_test_dir($cmd_class, $version);
```

Comment:
In the past we've tended to move files to the `$GENOME_TEST_INPUTS` when they're too big to include in the repository.  It seems better if the test has fewer external dependencies.  

If the trouble is we need these to be network accessible, maybe a `__FILE__.".d"`-type directory would be appropriate?